packages/core/src/types/IImage.ts

```diff
@@ -113,6 +113,12 @@ interface IImage {
     modalityLUT?: unknown;
     voiLUT?: CPUFallbackLUT;
   };
+
+  // Indicator if the image is complete, that is fully loaded.
+  complete?: boolean;
```

Comment:
ICachedImage has `status` which has `done`, then here we also have `complete` which is redundant and confusing. Can you please resolve this to only have one? maybe on the IImage?

Reply:
Left the one in IImage only